go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/filecoin-project/go-fil-markets v1.19.0
 	github.com/filecoin-project/go-jsonrpc v0.1.5
 	github.com/filecoin-project/go-state-types v0.1.3
-	github.com/filecoin-project/lotus v1.14.0
+	github.com/filecoin-project/lotus v1.14.2
```

Comment:
are we upgrading to 1.14.2?
Does the PR title need a correction?

Reply:
Done